custom_components/alexa_media/manifest.json

```diff
@@ -6,7 +6,7 @@
   "documentation": "https://github.com/custom-components/alexa_media_player/wiki",
   "issue_tracker": "https://github.com/custom-components/alexa_media_player/issues",
   "dependencies": ["persistent_notification", "http"],
-  "codeowners": ["@keatontaylor", "@alandtse"],
-  "requirements": ["alexapy==1.25.1", "packaging>=20.3", "wrapt~=1.12.1"],
```

Comment:
Why was this change reverted? It is now broken again on the latest Home Assistant
packaging>=20.3

Was this intentional? OR did you only intend to change alexapy version?

Reply:
see https://github.com/custom-components/alexa_media_player/pull/1353

Reply:
I had to edit wrapt>=1.12.1 this time as well since home assistant comes with a newer version of that as well now

Reply:
Oh, that shouldn't have been reverted. Not sure how that happened. Do you mind submitting a PR?

Reply:
yea I can

Reply:
Oh I already did it.

Reply:
great. Did you get wrapt also?

Reply:
Looks like you did. Thanks again ;)